src/pages/index.js

```diff
@@ -55,7 +55,7 @@ const IndexPage = () => (
         </Question>
         <Question>
           <h2>2. How can I join the community?</h2>
-          <p>You can join the community by <Link to="/login">clicking here</Link>. We have a Discord server for which, an invite link will be sent to your email. You will only need a Github account to join the community.</p>
+          <p>You can join the community by <Link to="/login">clicking here</Link>. We have a Discord server for which, an invite link will be sent to your email. You will only need a GitHub account to join the community.</p>
```

Comment:
Can you also change the `<Link>` tag to anchor tag with discord server link? I forgot to update it there.